src/notebooks_service.py

```diff
@@ -34,6 +34,8 @@
 from flask import send_from_directory
 from jupyterhub.services.auth import HubOAuth
 
+from kubernetes import client, config
```

Comment:
there should be a try/except so that it can work with a docker-compose deployment also